src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -438,12 +439,12 @@ Profile Profile::FromJson(const Json::Value& json)
 
 void Profile::SetFontFace(std::wstring fontFace) noexcept
 {
-    _fontFace = fontFace;
+    _fontFace = std::move(fontFace);
 }
 
 void Profile::SetColorScheme(std::optional<std::wstring> schemeName) noexcept
 {
-    _schemeName = schemeName;
+    _schemeName = std::move(schemeName);

```

Comment:
Out of curiosity, this works with a std::optional too?

Reply:
Yes.  Blurb from [cppreference](https://en.cppreference.com/w/cpp/utility/optional/optional). 

 >  Move constructor: If other contains a value, initializes the contained value as if direct-initializing (but not direct-list-initializing) an object of type T with the expression std::move(*other) and does not make other empty: a moved-from optional still contains a 
value, but the value itself is moved from. If other does not contain a value, constructs an object that does not contain a value. This constructor does not participate in overload resolution unless std::is_move_constructible_v<T> is true. It is a trivial constructor if std::is_trivially_move_constructible_v<T> is true.

So in this case  std::move() will cast the variable (schemeName) to an rvalue, allowing it to bind to a move constructor.